comps/reranks/README.md

```diff
@@ -0,0 +1,39 @@
+# ReRank Microservice
+
+## Introduction
+
+Provides functionality for reranking outputs.
+It allows for fine-tuning the results based on specific criteria, improving the overall quality and relevance for the user.
```

Comment:
Might be better to copy some text from one of the uServices like TEI. That nicely describes prioritizing documents.
For example...
```suggestion
Reranking Microservices, are straightforward yet immensely potent tools for semantic search. When provided with a query and a collection of documents, reranking swiftly indexes the documents based on their semantic relevance to the query, arranging them from most to least pertinent. Each microservice significantly enhances overall accuracy. In a text retrieval system, either a dense embedding model or a sparse lexical search index is often employed to retrieve relevant text documents based on the input. However, a reranking model can further refine this process by rearranging potential candidates into a final, optimized order.```